cmd/ttn-lw-cli/commands/users.go

```diff
@@ -60,6 +68,19 @@ func getUserID(flagSet *pflag.FlagSet, args []string) *ttnpb.UserIdentifiers {
 	return &ttnpb.UserIdentifiers{UserID: userID}
 }
 
+func getReceiverID(flagSet *pflag.FlagSet, args []string) *ttnpb.UserIdentifiers {
+	var userID string
+	if len(args) > 0 {
+		userID = args[1]
```

Comment:
Potential panic if `len(args) == 1`

Also, you're using this in combination with `getUserID()` which warns if there are multiple identifiers passed, but that seems to be intended for `transfer-owner-rights`.